src/backend/cuda/CMakeLists.txt

```diff
@@ -13,6 +13,13 @@ dependency_check(CUDA_FOUND "CUDA not found.")
 find_cuda_helper_libs(nvrtc)
 find_cuda_helper_libs(nvrtc-builtins)
 find_cuda_helper_libs(cudnn)
+get_filename_component(__found_cudnn_root ${CUDA_cudnn_LIBRARY} PATH)
+find_path(CUDA_cudnn_INCLUDE_DIR
```

Comment:
cool. Should nt we add 'required' somewhere ?

Reply:
The `REQUIRED` keyword works for only `find_package`. There is not official Find module for CUDNN from what I found from cmake docs. I am adding a check based on outcome of `find_path` , that should do the job. Have you tried the change on your system ?

Reply:
okok tks